commands/FBPrintCommands.py

```diff
@@ -424,3 +425,60 @@ def run(self, arguments, options):
       actionsDescription = fb.evaluateExpressionValue('(id)[{actions} componentsJoinedByString:@", "]'.format(actions=actions)).GetObjectDescription()
 
       print '{target}: {actions}'.format(target=targetDescription, actions=actionsDescription)
+
+class FBPrintAsCurl(fb.FBCommand):
+  def name(self):
+    return 'pcurl'
+
+  def description(self):
+    return 'Print the NSURLRequest (HTTP) as curl command.'
+
+  def options(self):
+    return [
+      fb.FBCommandArgument(short='-p', long='--portable', arg='portable', boolean=True, default=False, help='Embed request data as base64.'),
```

Comment:
"portable" is definitely opaque to me. I'm good with `--base64`. Other suggestions: `--inline(-data)`, `--embed(-data)`…